_data/projects.yml

```diff
@@ -565,3 +566,29 @@
   software_maturity: ["https://img.shields.io/badge/Partially%20met-orange.svg", "Partially met"]
   python3: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
   license: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
+
+- name: PyCDFpp
+  code: https://github.com/SciQLop/CDFpp
+  description: "A fast and easy to use C++ CDF library with Python bindings."
+  docs: https://pycdfpp.readthedocs.io/en/latest/
+  contact: Alexis Jeandet
+  keywords: ["heliophysics", "space physics", "magnetospheric physics", "CDF", "data"]
```

Comment:
Please redo these keywords as they must come from PyHC's [taxonomy](https://github.com/heliophysicsPy/heliophysicsPy.github.io/blob/main/_data/taxonomy.yml). If you feel the taxonomy is missing something, you may add to it in this PR.